README.md

```diff
@@ -307,9 +308,112 @@ t.resolve('3')
 t.close_requests_session()
 ```
 
+### Bugzilla
+
+Currently, Bugzilla supports Kerberos as well as authentication with 
+general login name and password. In the code general login will allow 
+you to retrieve a token that can be used as authentication for 
+subsequent API calls. For more details, 
+see : http://bugzilla.readthedocs.io/en/latest/api/index.html
+Below is the scenario which has been used while passing the general
+username and passowrd setting the authentication method as auth=rest.
+For kerberos you have to pass the parameter for auth=kerberos while 
+keeping other values to NONE.
+
+```python
+>>> from TicketUtil import RTTicket
+>>> t = BugzillaTicket('<bugzilla_url>','<product_name>', 'None', 'rest', username, password)
+```
+
+You should see the following response:
+```python
+INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): <bugzilla_url>
+INFO:root:Successfully authenticated to Bugzilla with token: <token-id>
+
+```
+You now have a BugzillaTicket object that is associated with the <product_name>.
+Some example workflows are found below. Notice that they all start with creating a BugzillaTicket object associated with a project (and with a ticket id when updating / resolving tickets). The last step is always closing the Requests session with t.close_requests_session().
+Check the details call below and execute accordingly in python:
+
+#### Create Bugzilla ticket
+```python
+import TicketUtil
+
+# Create a ticket object and pass the url, product_name as strings additionally set the environment variable for the username and password.
+username = os.environ.get("Username", "<email-address>")
```

Comment:
You can leave lines 343 / 344 out of the README. A user would understand that they could set username and password as environment variables. Or they may want to pass their credentials in directly when they create the BugzillaTicket object.